src/SearchField.svelte

```diff
@@ -18,24 +21,61 @@
   function onBlur() {
     dispatch("closed");
     inputValue = "";
+    currentBackspaceCount = 0;
+    setElFeedback('')
   }
 
   function onFieldBlur() {
     if (window.commandPalIgnoreBlur) return;
     onBlur();
   }
 
+  function setElFeedback(message) {
+    if (! inputElFeedback) {
+      inputElFeedback = document.getElementById(
+	inputName + "-feedback")
+    }
+
+    inputElFeedback.innerText = message
+    if (!!message) {
+      inputElFeedback.classList.add('open');
+    } else {
+      inputElFeedback.classList.remove('open');
+    }
+  }
+
   function onKeyDown(e) {
-    const keyCode = e.code.toLowerCase();
+    const keyCode = e.key.toLowerCase();
     if (keyCode === "enter") {
       dispatch("enter", inputValue);
       onBlur();
     } else if (keyCode === "arrowdown") {
+      currentBackspaceCount = 0;
+      setElFeedback('')
       dispatch("arrowdown");
     } else if (keyCode === "arrowup") {
+      currentBackspaceCount = 0;
+      setElFeedback('')
       dispatch("arrowup");
+    } else if (keyCode === "backspace" && backspaceCloseCount) {
+      // empty input: undefined if just opened, if keys added/deleted is ''
+      if (inputValue === undefined || inputValue === '' ){
```

Comment:
Might as well just simplify here to the [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) value
```suggestion
      if (!inputValue){
```